defs.bzl

```diff
@@ -12,8 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-load("@rules_jvm_external//:coursier.bzl", "coursier_fetch")
```

Comment:
Did you need to remove this because stardoc couldn't handle it? What was the error?

Reply:
Yep: https://github.com/bazelbuild/skydoc/issues/174

Reply:
That's a bummer, I'll look into the bug. In the meantime, I'm not sure it's a good idea to remove the repo name. There are scenarios in which if the repository is used as an external repo (which this one always will be), if you don't fully qualify the load statements with the repository's name, then things break. I'm blanking on when that is, but it is definitely a recommendation to keep the fully qualified names. 

I might suggest checking in the generated docs.md as a separate PR and keep this PR open to add the actual stardoc target later (so that the @rules_jvm_external doesn't need to be removed). It's theoretically a little messy because it's easier for the code to get out of sync, but I don't think there will be too much churn to the api in the very immediate future

Reply:
Sounds good to me. I'll make a new PR with the changes to the documentation.